onebusaway-android/src/main/java/org/onebusaway/android/ui/NavigationDrawerFragment.java

```diff
@@ -289,10 +283,10 @@ private void setSelectedNavDrawerItem(int itemId) {
     }
 
     @Override
```

Comment:
Ah, good catch.  No, you're right, we should change to `onAttach(Context context)`.  So this PR should be good as-is.  Thanks!